extensions/cornerstone/src/panels/PanelSegmentation.tsx

```diff
@@ -0,0 +1,286 @@
+
+export default function PanelSegmentation({
+  servicesManager,
+  commandsManager,
+  extensionManager,
+  configuration,
+  children,
+}: withAppTypes) {
+  const { segmentationService, viewportGridService, uiDialogService, customizationService } =
+    servicesManager.services;
+
+  const { segmentationsWithRepresentations, disabled } =
+    useActiveViewportSegmentationRepresentations({
+      servicesManager,
+    });
+
+  const handlers = {
+    onSegmentationAdd: async () => {
+      segmentationService.createLabelmapForViewport(viewportGridService.getActiveViewportId());
+    },
+
+    onSegmentationClick: (segmentationId: string) => {
+      segmentationService.setActiveSegmentation(
+        viewportGridService.getActiveViewportId(),
+        segmentationId
+      );
+    },
+
+    onSegmentAdd: segmentationId => {
+      segmentationService.addSegment(segmentationId);
+    },
+
+    onSegmentClick: (segmentationId, segmentIndex) => {
+      segmentationService.setActiveSegment(segmentationId, segmentIndex);
+      segmentationService.jumpToSegmentCenter(segmentationId, segmentIndex);
+    },
+
+    onSegmentEdit: (segmentationId, segmentIndex) => {
+      const segmentation = segmentationService.getSegmentation(segmentationId);
+
+      if (!segmentation) {
+        return;
+      }
+
+      const segment = segmentation.segments[segmentIndex];
+      const { label } = segment;
+
+      callInputDialog(uiDialogService, label, (label, actionId) => {
+        if (label === '') {
+          return;
+        }
+
+        segmentationService.setSegmentLabel(segmentationId, segmentIndex, label);
+      });
+    },
+
+    onSegmentationEdit: segmentationId => {
+      const segmentation = segmentationService.getSegmentation(segmentationId);
+
+      if (!segmentation) {
+        return;
+      }
+
+      const { label } = segmentation;
+
+      callInputDialog(uiDialogService, label, (label, actionId) => {
+        if (label === '') {
+          return;
+        }
+
+        segmentationService.addOrUpdateSegmentation(segmentationId, { label: label });
+      });
+    },
+
+    onSegmentColorClick: (segmentationId, segmentIndex) => {
+      const viewportId = viewportGridService.getActiveViewportId();
+      const color = segmentationService.getSegmentColor(viewportId, segmentationId, segmentIndex);
+
+      const rgbaColor = {
+        r: color[0],
+        g: color[1],
+        b: color[2],
+        a: color[3] / 255.0,
+      };
+      colorPickerDialog(uiDialogService, rgbaColor, (newRgbaColor, actionId) => {
+        if (actionId === 'cancel') {
+          return;
+        }
+
+        const color = [newRgbaColor.r, newRgbaColor.g, newRgbaColor.b, newRgbaColor.a * 255.0];
+        segmentationService.setSegmentColor(viewportId, segmentationId, segmentIndex, color);
+      });
+    },
+
+    onSegmentDelete: (segmentationId, segmentIndex) => {
+      segmentationService.removeSegment(segmentationId, segmentIndex);
```

Comment:
Ideally even really small ones like this would have their own commands module item so that it is possible to redirect/replace them